src/components/my-profile/JoinedMissions.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const JoinedMissions = () => {
+  const missions = useSelector((state) => state.missions);
+  const joinedMissions = missions.filter((mission) => mission.reserved);
```

Comment:
Please consider using the array's methods chaining, here you could simplify the filtering inside the `useSelector` directly

Reply:
done 😄 I agree, it's better that way 👍🏻